examples/dell5k_edge_system_iotcc.py

```diff
@@ -69,7 +69,7 @@ def read_disk_usage_stats():
 
 
 def read_network_bytes_received():
-    return round(net_stat.rx_tx_bytes('eth0')[0], 2)
+    return round(net_stat.rx_tx_bytes('ens33')[0], 2)
```

Comment:
Thanks for sharing reference, Piyush. `route | grep '^default' | grep -o '[^ ]*$'` command works with Linux OS.  I was looking for other solutions also.  We need to identify OS type and execute corresponding commands to get primary interface.

Since this is an example, can we get interface name from `sampleProp.conf` instead?